config.sample.yaml

```diff
@@ -321,6 +321,9 @@ ircService:
     # on instead of 113 here.
     # Default: 113.
     port: 1113
+    # The address to listen on for incoming ident requests.
+    # Default: 0.0.0.0
+    address: "::"
```

Comment:
`::` binds for both IPv6 and IPv4 on most operating systems (including Linux and macOS). In which case this works on a machine that doesn't have an IPv6 address.
Also see the conversation for [this pull request](https://github.com/matrix-org/synapse/pull/2232).